packages/easy-hydrator/composer.json

```diff
@@ -5,6 +5,8 @@
     "type": "symfony-bundle",
     "require": {
         "php": ">=7.2",
+        "phpstan/phpdoc-parser": "^0.4.9",
+        "rector/simple-php-doc-parser": "^0.8.21",
```

Comment:
Why is thai added? Seems unrelated

Reply:
i agree

ok, i will split it in another MR